WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import UIKit
+import SwiftUI
+
+/// View model for `HubMenu`.
+///
+final class HubMenuViewModel: ObservableObject {
+
+    let siteID: Int64
+
+    let storeTitle = ServiceLocator.stores.sessionManager.defaultSite?.name ?? Localization.myStore
+    var storeURL: URL {
+        guard let urlString = ServiceLocator.stores.sessionManager.defaultSite?.url, let url = URL(string: urlString) else {
+            return WooConstants.URLs.blog.asURL()
+        }
+
+        return url
+    }
+
+    /// Child items
+    ///
+    @Published private(set) var menuElements: [Menu] = []
```

Comment:
Yes. A trial candidate is currently working on integrating the `Analytics` menu, and it will be under a feature flag, so the menu elements are dynamic for this reason.